packages/contracts-ops/contracts/Config.sol

```diff
@@ -72,6 +72,16 @@ abstract contract Config {
 
     function governanceRouter(string memory domain)
         public
+        override
+        onlyInitialized
+        returns (GovernanceRouter)
+    {
+        return GovernanceRouter(address(governanceRouterUpgrade(domain).proxy));
+    }
+
+    function governanceRouterUpgrade(string memory domain)
```

Comment:
nit: should be above `governanceRouter` which uses this function

Reply:
https://github.com/nomad-xyz/monorepo/pull/484/commits/2d50e92c61b3ddbf5e85a9125d8204ef1ed675eb